modules/integration/src/test/scala/scala/cli/integration/BspTests3NextRc.scala

```diff
@@ -1,3 +1,95 @@
 package scala.cli.integration
 
-class BspTests3NextRc extends BspTestDefinitions with BspTests3Definitions with Test3NextRc
+import ch.epfl.scala.bsp4j as b
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.async.Async.{async, await}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.jdk.CollectionConverters.*
+import scala.util.Properties
+
+class BspTests3NextRc extends BspTestDefinitions with BspTests3Definitions with Test3NextRc {
+  test("BSP respects --cli-default-scala-version & --predefined-repository launcher options") {
+    // 3.5.0-RC1-fakeversion-bin-SNAPSHOT has too long filenames for Windows.
```

Comment:
wow just learning this also, apparently Microsoft say you can prefix with \\?\ to get a longer length limit https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry

Reply:
I'll try it in a separate PR later, nice!